Shokofin/LibraryScanner.cs

```diff
@@ -51,6 +51,11 @@ public bool ShouldIgnore(MediaBrowser.Model.IO.FileSystemMetadata fileInfo, Base
                     return false;
                 }
 
+                if (!fileInfo.IsDirectory && Plugin.Instance.IgnoredFolders.Contains(fileInfo.FullName.ToLowerInvariant())) {
```

Comment:
It would be better to check if it is a directory and if the directory name is in the list, then return `true` to ignore the directory and all it's entries.

Reply:
Also, forgot to say it earlier but `FullName` is the full path, not the entry name.